sdk/storage/azure-storage-blobs/NOTICE.txt

```diff
@@ -30,41 +30,43 @@ bring it to our attention. Post an issue or email us:
 
 The attached notices are provided for information only.
 
-------------------------------------------------------------------------------
 
+License notice for CMake Modules AddGoogleTest
 -------------------------------------------------------------------
 
-nlohmann/json e7b3b40b5a95bc74b9a7f662830a27c49ffc01b4 - MIT
-(c) 2016
-(c) Y Seomi
-Copyright 2014
-T68Oa (c) PiP6Le
-Si4i (c) AzZGz3vae
-Copyright 2015 Google Inc.
-Copyright 2018 Google Inc.
-Copyright Louis Dionne 2015
-Copyright (c) 2015 Max Woolf
-Copyright 2017 Roman Lebedev.
-Copyright (c) 2009 Google Inc.
-copyright' (c) 2013-2017 Niels
-Copyright (c) 2016 Nicolas Seriot
-Copyright (c) 2009 Florian Loitsch.
-copyright (c) 2013-2017 Niels Lohmann
-Copyright (c) 2013-2019 Niels Lohmann.
-Copyright (c) 2015-2017 Niels Lohmann.
-Copyright (c) 2016-2018 Viktor Kirilov
-Copyright (c) 2016-2019 Viktor Kirilov
-Copyright (c) 2018-2019 Bryan Gillespie
-Copyright 2016 Ismael Jimenez Martinez.
-Copyright (c) 2012, Erik Edlund <erik.edlund@32767.se>
-Copyright (c) 2018 Vitaliy Manushkin <agri@akamo.info>
-Copyright (c) 2013-2019 Niels Lohmann (http://nlohmann.me)
-Copyright (c) 2013-2019 Niels Lohmann <http://nlohmann.me>
-Copyright (c) 2009 Florian Loitsch (http://florian.loitsch.com/)
-Copyright (c) 2007 Free Software Foundation, Inc. <https://fsf.org/>
-Copyright (c) 2008-2009 Bjorn Hoehrmann (http://bjoern.hoehrmann.de/) <bjoern@hoehrmann.de>
-Copyright (c) 2008-2009 Bjoern Hoehrmann <bjoern@hoehrmann.de> sa http://bjoern.hoehrmann.de/utf-8/decoder/dfa
-Copyright (c) 1996 - 2020, Daniel Stenberg, <daniel@haxx.se>
+BSD 3-Clause License
+
+Copyright (c) 2017, University of Cincinnati
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+
+* Redistributions of source code must retain the above copyright notice, this
```

Comment:
IIRC, the copyright lines above ("Copyright (C) ...") should remain intact; and even maybe if we contribute, MS notice should go there too.

Reply:
Correct, the NOTICE.txt needs to be an unedited and complete copy of the exact license.  That includes spelling errors must not be fixed if they exist in the original license.